provider/pipeline_jenkins_stage.go

```diff
@@ -96,5 +105,16 @@ func (s *jenkinsStage) SetResourceData(d *schema.ResourceData) error {
 	if err != nil {
 		return err
 	}
-	return d.Set("property_file", s.PropertyFile)
+	err = d.Set("property_file", s.PropertyFile)
+	if err != nil {
+		return err
+	}
+
+	// Coveo specific
+	err = d.Set("pipeline_metadata", s.PipelineMetadata)
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
(Nitpick) Idem to previous comment
```suggestion
	return d.Set("pipeline_metadata", s.PipelineMetadata)
```